forth.fth

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env gforth
```

Comment:
Huh, I tested it on Arch Linux 2017.02.01 and this shebang worked, which it probably shouldn't have.